Workbooks/InsecureProtocols.json

```diff
@@ -63,7 +60,7 @@
             "crossComponentResources": [
               "{Subscription}"
             ],
-            "value": "/subscriptions/44e4eff8-1fcb-4a22-a7d6-992ac7286382/resourceGroups/SOC/providers/Microsoft.OperationalInsights/workspaces/CyberSecurityDemo",

```

Comment:
please replace this line to include genetic parameters for subscription/ResourceGroup/workspace :

"value": "/subscriptions/<subs_ID>/resourceGroups/<rg_name>/providers/Microsoft.OperationalInsights/workspaces/<workspace_name>",

Reply:
This line is be removed completely and is from the existing published version.